File: Questao3/resposta.py
#Escreva um programa que dado um array, some todos os seus elementos e imprima da tela.
arr = [2, 18, 98, 18, 96, 97, 71, 34, 11, 4, 4, 36, 21, 17 , 5, 44, 30, 12, 93, 2, 15, 6, 90, 29, 48]
sum = 0

for i in range(0, len(arr)):  
   sum = sum + arr[i];

print("a soma dos elementos nesse array é : " + str(sum))

File: Questao2/resposta.js
// Escreva um programa que, dado um array, gere um novo array com todos os múltiplos de 3 do array inicial


let arrayInicial = [17, 6, 6, 8, 4, 79, 5, 30, 4, 3, 2, 1, 1, 7, 10, 3, 42, 2, 2, 9, 90, 4 ]
for(let i = 0; i < arrayInicial.length; i++) {
    if(arrayInicial[i] % 3 === 0){
       console.log(arrayInicial[i] + " é um múltiplo de 3" )
    }
  
}

